server/src/main/java/org/elasticsearch/monitor/os/OsStats.java

```diff
@@ -500,7 +500,7 @@ public Cgroup(
         @Override
         public void writeTo(final StreamOutput out) throws IOException {
             out.writeString(cpuAcctControlGroup);
-            out.writeLong(cpuAcctUsageNanos);
+            out.writeString(cpuAcctUsageNanos);
```

Comment:
This will definitely see a new TransportVersion. See `TransportVersions.java`

Reply:
Thanks for highlighting this. I have added a new TransportVersion.